File: Programming Basics with Python/Първи стъпки в програмирането/food_delivery.py

chicken = int(input())
fish = int(input())
vegan = int(input())
delivary = 2.50

chicken_menu = chicken * 10.35
fish_menu = fish * 12.40
vegan_menu = vegan * 8.15

menu_price = chicken_menu + fish_menu + vegan_menu
dessert_price = menu_price * (20/100)
total = menu_price + dessert_price + delivary

print(total)

File: Programming Basics with Java/Първи стъпки в програмирането/FoodDelivery.java
import java.util.Scanner;

public class FoodDelivery 
{
    public static void main(String[] args) 
    {
        Scanner input = new Scanner(System.in);

        int chicken = input.nextInt();
        int fish = input.nextInt();
        int vegan = input.nextInt();

        double sum = (chicken * 10.35) + (fish * 12.4) + (vegan * 8.15);
        double desert = sum * 0.2;
        double total = sum + desert + 2.5;

        System.out.println(total);
    }
}
